go.mod

```diff
@@ -13,7 +13,6 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/cobra v1.2.1
-	github.com/spf13/viper v1.8.1
```

Comment:
This is from me removing the `kvcli`, `kvd` directories and not cleaning of up mods